components/src/molecules/WizardHeader/index.tsx

```diff
@@ -45,16 +46,23 @@ export const WizardHeader = (props: WizardHeaderProps): JSX.Element => {
           currentStep != null &&
           totalSteps != null &&
           currentStep > 0 ? (
-            <LegacyStyledText className={styles.step_text}>
+            <StyledText
+              desktopStyle="bodyDefaultRegular"
+              oddStyle="bodyTextSemiBold"
+              color={grey60}
+            >
               {`Step ${currentStep} / ${totalSteps}`}
-            </LegacyStyledText>
+            </StyledText>
           ) : null}
         </Flex>
         {onExit != null ? (
           <Btn onClick={onExit} aria-label="Exit" disabled={exitDisabled}>
```

Comment:
```suggestion
          <Btn
            onClick={onExit}
            aria-label="Exit"
            disabled={exitDisabled}
            className={styles.exit_button}
          >
```


https://github.com/user-attachments/assets/d8dd7a64-79fb-430f-8fa8-715f64f1e234